p2p/transport/websocket/websocket_test.go

```diff
@@ -548,3 +548,71 @@ func TestResolveMultiaddr(t *testing.T) {
 		})
 	}
 }
+
+func TestListenerResusePort(t *testing.T) {
+	laddr := ma.StringCast("/ip4/127.0.0.1/tcp/5002/ws")
```

Comment:
Can you listen on an unspecified port instead?
```go
ma.StringCast("/ip4/127.0.0.1/tcp/0/ws")
```

That way, this test won't fail it some other application is listening on port 5002.

Reply:
Sure, will make this change.